designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -650,6 +650,9 @@ fun SquooshRoot(
             }
             else -> {}
         }
+    }
+    val scrollState = rememberScrollableState { delta ->
```

Comment:
`scrollState` is still memoized, so the `LaunchedEffect` won't get re-run even though a new value of `setScrollState` is computed every run through.